Sources/CocoaLumberjack/include/DDTTYLogger.h

```diff
@@ -31,7 +31,7 @@
     static inline DDColor* DDMakeColor(CGFloat r, CGFloat g, CGFloat b) {return [DDColor colorWithRed:(r/255.0f) green:(g/255.0f) blue:(b/255.0f) alpha:1.0f];}
 #elif defined(DD_CLI) || !__has_include(<AppKit/NSColor.h>)
     // OS X CLI
-    #import <CocoaLumberjack/CLIColor.h>
+    #import "CLIColor.h"
```

Comment:
Was this import change on purpose? It's not relevant for SPM and is inside a public header. So it should use angle bracket style, right?

Reply:
@ffried 
It was one of the steps in experiment that I forget to reverse :)